epytope/test/TestSpacerDesign.py

```diff
@@ -30,10 +30,10 @@ def test_standart_functions(self):
         epi_pred = EpitopePredictorFactory("Syfpeithi")
         cl_pred = CleavageSitePredictorFactory("PCM")
 
-        sbws = EpitopeAssemblyWithSpacer(self.epis, cl_pred, epi_pred, self.alleles, solver="cbc")
+        sbws = EpitopeAssemblyWithSpacer(self.epis, cl_pred, epi_pred, self.alleles, solver="glpk")
```

Comment:
That's still something which has to be decided (`cbc` or `glpk`)

Reply:
@b-schubert

Reply:
Let's use `glpk`, could you please write it down? We should write a proper changelog anyway at some point.